### DISCUSSION TITLE
Question about .cool file bias vector values in regions with 0 contacts

### DISCUSSION THREAD

**Discussion by kalavattam** [open2c/cooler#349] [Q&A] [answered]
I have received some `.cool` files for my work and I want to ensure they are formatted correctly. How are regions with 0 contacts represented in the bias vector? Should they be NaN, 0, or some positive finite value? Thank you.

**Comment by nvictus** [comment, marked as answer]
Hi Kris! Regions with 0 contacts are normally masked out for matrix balancing (the algorithm would never converge if they were kept). Masked/filtered bins are normally encoded as NaN in the weight vector, which will "NaN-out" the corresponding row/column of the matrix when the weights are applied (B_ij = w_i * w_j * A_ij).
